content/packages/ally-tiktok.yml

```diff
@@ -0,0 +1,15 @@
+name: ally-tiktok
+description: An TikTok driver for AdonisJS Ally
```

Comment:
You already send that package : https://github.com/adonisjs-community/adonis-packages/blob/main/content/packages/ally-tiktok.yml